snapcraft/script/bridge.start

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+logger -t "conjure-up/systemd" "[DEBUG] Creating conjure-up network bridge"
 ip link add dev conjureup0 type bridge
 ip addr add 10.99.0.1/24 dev conjureup0
 ip link set dev conjureup0 up
 
+sleep 5
+
```

Comment:
seems error prone, is there any thing we can loop on to be sure we've waited long enough?